QGCSetup.pri

```diff
@@ -121,12 +121,15 @@ LinuxBuild {
         libicui18n.so* \
         libQt5TextToSpeech.so.5
 
-    !contains(DEFINES, __rasp_pi2__) {
-        # Some Qt distributions link with *.so.56
-        QT_LIB_LIST += \
-            libicudata.so.56 \
-            libicui18n.so.56 \
-            libicuuc.so.56
+        exists($$[QT_INSTALL_LIBS]/libicudata.so.56) {
```

Comment:
I think this should be either just $$QT_INSTALL_LIBS, or $${QT_INSTALL_LIBS}. Something seems to be confusing qmake breaking some variable expansion. This is just a guess.